utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -122,8 +124,92 @@ class WriteBatchEntryComparator {
   const ReadableWriteBatch* write_batch_;
 };
 
+typedef SkipList<WriteBatchIndexEntry*, const WriteBatchEntryComparator&>
+    WriteBatchEntrySkipList;
+
+class WBWIIteratorImpl : public WBWIIterator {
```

Comment:
I see this has moved from the `.c` to the `.h` file. I am wondering if we should have just moved the definition and kept the declaration in the `.cc` file? I don't know the answer, I just thought I would query it...

Reply:
I am fine with putting only the definition in the header file.  I did a copy/paste move.  There does not seem to be a good standard for RocksDB on what goes in header versus source files.

Reply:
Yeah indeed I have seen a mix of styles in the RocksDB code-base. I wasn't sure what the "house style" was supposed to be though...